src/lang/es.json

```diff
@@ -140,7 +140,7 @@
   "Add FTP chart": "Añadir gráfico de FTP",
   "Add FTP/kg": "Añadir FTP/kg",
   "Add FTP/kg chart": "Añadir gráfico FTP/kg",
-  "Add Gear": "Añadir Equipo",
+  "Add Gear": "Añadir equipamiento",
```

Comment:
For the record, in Spanish, capitalization follows standard grammar rules - we don't capitalize every word in titles or buttons like in English.